src/Nethermind/Nethermind.Blockchain.Test/Receipts/PersistentReceiptStorageTests.cs

```diff
@@ -24,6 +24,7 @@ public class PersistentReceiptStorageTests
         [SetUp]
         public void SetUp()
         {
+            // Warning Ropsten Network is Deprecated
             RopstenSpecProvider specProvider = RopstenSpecProvider.Instance;
```

Comment:
I mean in every place we could change `RopstenSpecProvider.Instance; `to `MainnetSpecProvider.Instance;`
and remove `RopstenSpecProvider.cs` class